new_builder_evaluation/app/api/endpoints/evaluation_decentralized.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) MLCommons and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from fastapi import APIRouter
+from pydantic import BaseModel
+
+from app.domain.evaluation_decentralized import Evaluation
+
+
+class ModelSingleInput(BaseModel):
+    text: str
+
+
+router = APIRouter()
+
+
+@router.get("/test_des")
+async def test_des():
+    model = Evaluation()
+    api = model.test_eval(
+        "sentiment", "models/sentiment/123-julian_sentiment.zip", 3333
```

Comment:
Should we change this to make it usable in general? Or is there a specific reason for this values to be hardcoded?